coalib/bearlib/languages/__init__.py

```diff
@@ -3,40 +3,56 @@
 """
 
 # Start ignoring PyUnusedCodeBear
-from .Language import Language
-from .Language import Languages
-
-from .definitions.Unknown import Unknown
 from .definitions.Bash import Bash
+
 from .definitions.C import C
 from .definitions.CPP import CPP
 from .definitions.CSharp import CSharp
 from .definitions.CSS import CSS
+
```

Comment:
I don't think there is any need for spaces like these. You may want to remove them.